glue_job.tf

```diff
@@ -252,8 +252,19 @@ data "aws_vpc" "main" {
   id = var.vpc_id
 }
 
+data "aws_route_tables" "all_route_tables" {
```

Comment:
My first attempt was to look up the route table associated with `var.glue_physical_connection_requirements.subnet_id`, but `aws_route_tables` and `aws_route_table` data resources don't return the default route table when it's implicitly defined (see https://github.com/hashicorp/terraform-provider-aws/issues/12099 for a discussion on this).